<launch>
    <node name="line2single" pkg = "line2single" type = "line2single" output = "screen">
        <param name = "robotName" type = "str" value = "" />
        <param name = "localHostIp" type = "str" value = "172.19.221.5" />
        <param name = "unrealIp" type = "str" value = "172.19.208.1" />
        <param name = "dataPort" type = "str" value = "7050" />
        <param name = "raisimServerPort" type = "int" value = "8080" />
        <param name = "rosbagPath" type = "str" value = "/home/siris/catkin_ws/rosbags/line_motion-20240302.bag" />
    </node>
</launch>